spec/solitaire_spec.rb

```diff
@@ -1,7 +1,29 @@
 require 'spec_helper'
 require_relative '../solitaire'
 
-describe Solitaire, '#decrypt_message' do
+describe Solitaire, '#prepare(message)' do
+  it "discards non A-Z letters, groups in 5's, adds X's" do
+    message = 'Code in Ruby, live longer!'
+    expected_output = ["CODEI", "NRUBY", "LIVEL", "ONGER"]
+
+    prepared_message  = Solitaire.new.prepare(message)
+
+    expect(prepared_message).to eq expected_output
+  end
+end
+
+xdescribe Solitaire, '#encrypt_message' do
+  it 'successfully encrypts a message' do
+    message = 'Code in Ruby, live longer!'
+    expected_output = 'GLNCQ MJAFF FVOMB JIYCB'
+
+    encrypted_message = Solitaire.new.encrypt_message(message)
+
+    expect(encrypted_message).to eq expected_output
+  end
+end
```

Comment:
These tests look okay, although I would move the message argument into the constructor instead of as a parameter into the public method.